### PR TITLE
[maui-aiassistview] Added stop responding view style keys

### PR DESCRIPTION
**Pull request by RavikumarAkuthotaSf4250**
Added stop responding view style keys in Themes and Assistview Styles topics

### CODE REVIEW

File: MAUI/AIAssistView/styles.md [hunk 1]
```diff
@@ -952,5 +952,87 @@ To style the send item button based on its state, set values to the in-built key
...
+To style the stop responding view based on its appearance, set values to the in-built keys in the resource dictionary.
+
+<table> 
+<tr> 
+<th> Key </th>
+<th> Description </th> 
+</tr> 
+<tr> 
+<td> SfAIAssistViewStopRespondingIconColor </td>
+<td> Icon color for the stop responding action. </td>
```
Comment (Lakshminatarajan) [inline MAUI/AIAssistView/styles.md:969]: Icon color for the stop responding view.

File: MAUI/AIAssistView/styles.md [hunk 2]
```diff
@@ -952,5 +952,87 @@ To style the send item button based on its state, set values to the in-built key
...
+<th> Key </th>
+<th> Description </th> 
+</tr> 
+<tr> 
+<td> SfAIAssistViewStopRespondingIconColor </td>
+<td> Icon color for the stop responding action. </td> 
+</tr> 
+<tr> 
+<td> SfAIAssistViewStopRespondingTextColor </td> 
+<td> Text color for the stop responding action. </td>
```
Comment (Lakshminatarajan) [inline MAUI/AIAssistView/styles.md:973]: Text color for the stop responding view.

File: MAUI/AIAssistView/styles.md (hunk 1 above)
Comment (RavikumarAkuthotaSf4250) [reply MAUI/AIAssistView/styles.md:969]: Updated akka

File: MAUI/AIAssistView/styles.md (hunk 2 above)
Comment (RavikumarAkuthotaSf4250) [reply MAUI/AIAssistView/styles.md:973]: Updated to **view**

File: MAUI/AIAssistView/styles.md [hunk 3]
```diff
@@ -952,5 +952,87 @@ To style the send item button based on its state, set values to the in-built key
...
+<td> SfAIAssistViewStopRespondingFontSize </td> 
+<td> Font size of the stop responding text. </td> 
+</tr> 
+<tr> 
+<td> SfAIAssistViewStopRespondingFontAttributes </td> 
+<td> Font attributes used for the stop responding text. </td>
+</tr>
+<tr> 
+<td> SfAIAssistViewStopRespondingStroke </td> 
+<td> Stroke color for the stop responding action. </td>
```
Comment (Lakshminatarajan) [inline MAUI/AIAssistView/styles.md:989]: Stroke color for the stop responding view.

File: MAUI/AIAssistView/styles.md [hunk 4]
```diff
@@ -952,5 +952,87 @@ To style the send item button based on its state, set values to the in-built key
...
+<td> SfAIAssistViewStopRespondingFontAttributes </td> 
+<td> Font attributes used for the stop responding text. </td>
+</tr>
+<tr> 
+<td> SfAIAssistViewStopRespondingStroke </td> 
+<td> Stroke color for the stop responding action. </td> 
+</tr> 
+<tr> 
+<td> SfAIAssistViewStopRespondingStrokeThickness </td> 
+<td> Stroke thickness for the stop responding action. </td>
```
Comment (Lakshminatarajan) [inline MAUI/AIAssistView/styles.md:993]: Stroke thickness for the stop responding view.

File: MAUI/AIAssistView/styles.md (hunk 3 above)
Comment (RavikumarAkuthotaSf4250) [reply MAUI/AIAssistView/styles.md]: updated

File: MAUI/AIAssistView/styles.md (hunk 4 above)
Comment (RavikumarAkuthotaSf4250) [reply MAUI/AIAssistView/styles.md]: Updated to view

**Review by Lakshminatarajan — APPROVED** [review]
Approved

**Review by PiruthivirajM — APPROVED** [review]

**Pull request merged**